frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx

```diff
@@ -192,15 +192,18 @@ export const OperatorHubPage = withFallback(
         <div className="co-catalog">
           <PageHeading title="OperatorHub" />
           <p className="co-catalog-page__description">
-            Discover Operators from the Kubernetes community and Red Hat partners, curated by Red
-            Hat. You can purchase commercial software through{' '}
-            <ExternalLink
-              href="https://marketplace.redhat.com/en-us?utm_source=openshift_console"
-              text="Red Hat Marketplace"
-            />
-            . You can install Operators on your clusters to provide optional add-ons and shared
-            services to your developers. After installation, the Operator capabilities will appear
-            in the <Link to="/catalog">Developer Catalog</Link> providing a self-service experience.
+            <Trans ns="olm">
+              Discover Operators from the Kubernetes community and Red Hat partners, curated by Red
+              Hat. You can purchase commercial software through{' '}
+              <ExternalLink
+                href="https://marketplace.redhat.com/en-us?utm_source=openshift_console"
+                text="Red Hat Marketplace"
```

Comment:
link text `Red Hat Marketplace` is not translated
<img width="690" alt="Screen Shot 2020-12-16 at 4 51 09 PM" src="https://user-images.githubusercontent.com/12692381/102326185-09c2d900-3fbf-11eb-92ec-ad96bf73a152.png">

Reply:
@TheRealJon  I didn't see `Red Hat Marketplace` string is translated in new code